<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_6_1_naming_convention</class>
 <widget class="QDialog" name="dialog_6_1_naming_convention">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1253</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OFR Naming Convention</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1141</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="10">
     <widget class="QLabel" name="label_19">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>DOCUMENT TYPE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="10">
     <widget class="QComboBox" name="comboBox_6_type">
      <item>
       <property name="text">
        <string>GA: General admin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MD: Marketing</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FP: Fee proposal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LT: Letter</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DN: Design note</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OF: Outline strategy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DF: Detailed strategy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RF: Retrospective strategy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FA: Fire risk assessment</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FS: Fire survey report</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FN: File note</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MN: Meeting notes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CS: Calculation sheet</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SK: Sketch</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DW: Drawing</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>XO: Expert opinion</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_15">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>REVISION</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="12">
     <widget class="QComboBox" name="comboBox_7_security_status">
      <item>
       <property name="text">
        <string>CIC: Commercial in confidence</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WPC: Without prejudice and confidential</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SDS: Secure document</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FID: Free issue document (no security status)</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="comboBox_2_revision">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Q00: First issue for internal review</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Q01: Reviewer's comments</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Q02: Authoriser's comments</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D00: First issue to others for comment</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D01: Sub-sequent external reviews</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>R00: First issue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>R01: Second issue</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_16">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>DATE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QLabel" name="label_18">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>FILE TITLE (PLAIN ENGLISH)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="12">
     <widget class="QLabel" name="label_20">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SECURITY STATUS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="11">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLineEdit" name="lineEdit_4_project_stage">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>WP1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QLineEdit" name="lineEdit_5_title">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Outline_fire_strategy_report</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="lineEdit_3_project_no">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>OX2001</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_14">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PROJECT NO.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label_17">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PROJECT STAGE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit_1_date">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>20151021</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_result">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>71</y>
     <width>1141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>4</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>4</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>4</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_copy">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>10</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy file name</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_1_date</tabstop>
  <tabstop>comboBox_2_revision</tabstop>
  <tabstop>lineEdit_3_project_no</tabstop>
  <tabstop>lineEdit_4_project_stage</tabstop>
  <tabstop>lineEdit_5_title</tabstop>
  <tabstop>comboBox_6_type</tabstop>
  <tabstop>comboBox_7_security_status</tabstop>
  <tabstop>pushButton_copy</tabstop>
  <tabstop>lineEdit_result</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
